crates/weaver_forge/src/config.rs

```diff
@@ -260,9 +264,38 @@ impl CaseConvention {
                 });
                 conv.convert(&text)
             }
-            CaseConvention::PascalCase => text.to_case(Case::Pascal),
+            CaseConvention::PascalCase => {
+                // Convert to pascal case but do not consider digits
+                // as boundaries.
+                let conv = PASCAL_CASE.get_or_init(|| {
+                    Converter::new()
+                        .add_boundary(LowerUpper)
+                        .add_boundary(Acronym)
+                        .remove_boundary(DigitLower)
```

Comment:
I absolutely trust you that it works, but it'd be nice to add a test with a digit in the text